packages/react/src/components/NodeWrapper/index.tsx

```diff
@@ -155,6 +155,23 @@ export function NodeWrapper<NodeType extends Node>({
     }
   };
 
+  const onFocus = () => {
+    if (disableKeyboardA11y) {
+      return;
```

Comment:
We should also return early if the user clicks a node (because we only want to focus on tab). For this you could use:

`nodeRef.current?.matches(':focus-visible')`